packages/elements-core/src/components/TryIt/TryIt.tsx

```diff
@@ -197,81 +214,94 @@ export const TryIt: React.FC<TryItProps> = ({
     }
   };
 
-  const serversSelect = (
-    <Select
-      aria-label="Servers"
-      options={servers.map(server => ({ value: server.description || '' }))}
-      value={chosenServer?.description || ''}
-      size="sm"
-      onChange={(value: React.Key) => {
-        const server = servers.find(server => server.description === value);
-
-        setChosenServer(server);
-      }}
-    />
-  );
-
-  return (
-    <Box rounded="lg" overflowY="hidden">
-      <Panel isCollapsible={false} p={0} className="TryItPanel">
-        <Panel.Titlebar rightComponent={servers.length > 1 ? serversSelect : undefined} bg="canvas-300">
-          <Box role="heading" fontWeight="bold">
-            <Text color={!isDark ? HttpMethodColors[httpOperation.method] : undefined}>
-              {httpOperation.method.toUpperCase()}
-            </Text>
-            <Text ml={2}>{httpOperation.path}</Text>
-          </Box>
-        </Panel.Titlebar>
+  const isOnlySendButton =
+    !httpOperation.security?.length && !allParameters.length && !formDataState.isFormDataBody && !mediaTypeContent;
 
+  const tryItPanelContents = (
+    <>
+      {httpOperation.security?.length ? (
         <TryItAuth
           onChange={setOperationAuthValue}
           operationSecurityScheme={httpOperation.security ?? []}
```

Comment:
Kind of awkward that we are checking if `httpOperation.security` is defined (and even not empty!), but then we have to provide a fallback anyway.

Reply:
Don't need the fallback anymore - will remove